File: backend/src/routes/acompanhamentoPostesRoutes.js
const express = require('express');
const router = express.Router();
const acompanhamentoPostesController = require('../controllers/acompanhamentoPostesController');

// Buscar status de anos desligados/ocultos do Postes para um cliente
router.get('/:cnpj', acompanhamentoPostesController.getPostesStatus);

// Salvar status de anos desligados/ocultos do Postes para um cliente
router.post('/:cnpj', acompanhamentoPostesController.setPostesStatus);

module.exports = router;

File: python_backend/routes/acompanhamento_postes_routes.py
from fastapi import APIRouter, Path
from controllers import acompanhamento_postes_controller

router = APIRouter()

@router.get('/{cnpj}')
async def get_postes_status(cnpj: str = Path(...)):
    return await acompanhamento_postes_controller.get_postes_status(cnpj)

@router.post('/{cnpj}')
async def set_postes_status(cnpj: str = Path(...), payload: dict = None):
  return await acompanhamento_postes_controller.set_postes_status(cnpj, payload or {})
